data/transactions/logic/TEAL_opcodes.md

```diff
@@ -1154,6 +1154,15 @@ bitlen interprets arrays as big-endian integers, unlike setbit/getbit
 - **Cost**: 10
 - LogicSigVersion >= 4
 
+## sha3_256
+
+- Opcode: 0x96
```

Comment:
For the new ecdsa there was a gap in the op_codes which they filled. I see for the PR https://github.com/algorand/go-algorand/pull/3218, which is a new OP code for for ed25519 verification they just select an unused opcode, I think they did the same for the new math opcodes.

Reply:
Ah ok, this should be good then, thanks for looking!

Reply:
We do try to leave them where they are.  FYI, 0x96 is going to be taken in the next release by `bsqrt`.  I think we're considering the pull for SHA3-256 now, but it would probably be in the following TEAL version.  If compatibility with that matters, maybe move this to 0x9f for now, which saves a little room for us in case another opcode or two goes in.

But then again, perhaps you're just prototyping with this, and you wouldn't mind a change whevever we figure out a spot for this.

Reply:
Hi, we're happy to wait and then to change the opcode when you have figured out a spot for it.